bee-message/src/unlock_block/signature.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{signature::Signature, Error};
+
+use bee_common::packable::{Packable, Read, Write};
+
+/// An [`UnlockBlock`](crate::unlock_block::UnlockBlock) which is used to unlock a signature locked
+/// [`Input`](crate::input::Input).
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
+#[cfg_attr(feature = "serde1", derive(serde::Serialize, serde::Deserialize, derive_more::From))]
+pub struct SignatureUnlockBlock(Signature);
+
+impl SignatureUnlockBlock {
+    /// The [`UnlockBlock`](crate::unlock_block::UnlockBlock) kind of a [`SignatureUnlockBlock`].
+    pub const KIND: u8 = 0;
+
+    /// Creates a new [`SignatureUnlockBlock`].
+    pub fn new(signature: Signature) -> Self {
+        Self(signature)
+    }
+
+    /// Returns the actual [`Signature`] of the [`SignatureUnlockBlock`].
+    pub fn signature(&self) -> &Signature {
+        &self.0
+    }
+}
+
+impl core::ops::Deref for SignatureUnlockBlock {
+    type Target = Signature;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
```

Comment:
Can `derive_more`, which we use in other places already, derive `Deref` for these simple type wrappers?